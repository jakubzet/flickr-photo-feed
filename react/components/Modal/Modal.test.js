import React from "react";
import { render } from "../../utils/reactTestingLibrary";
import { Modal } from "./Modal";

describe("Modal", () => {
  it("Renders", () => {
    const { container, asFragment } = render(<Modal />);

    expect(container).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });
});
