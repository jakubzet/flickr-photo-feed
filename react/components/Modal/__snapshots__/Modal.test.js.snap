// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Renders 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa lcDbZB"
  >
    <div
      class="sc-htpNat gznbeD"
    >
      <div
        class="sc-bwzfXH hZfRXl"
      />
    </div>
  </div>
</DocumentFragment>
`;
